src/app/component/MissionGroup/MissionGroup.jsx

```diff
@@ -55,6 +55,14 @@ const MissionGroup = ({ callToAction, group, groupCallToAction, showViewRoute })
   };
   const onClickViewRouteButton = () => {
     console.log("View Routes");
+    missions.forEach((mission) => {
+      console.log(mission);
```

Comment:
We need to remove debugging console logs

Reply:
No longer there